base/context.go

```diff
@@ -113,18 +113,14 @@ func (ctx *Context) GetClientTLSConfig() (*tls.Config, error) {
 	}
 
 	if ctx.Certs != "" {
-		if log.V(1) {
```

Comment:
Upgrading these from `V(1)` to default will be irritating. It might be better to remove them than to promote them.